toolkits/springer/packages/springer-webfonts/HISTORY.md

```diff
@@ -1,3 +1,6 @@
+## 1.0.0 (2019-02-10)
```

Comment:
2020 ??

Reply:
Oh. I wonder how many packages I've done that for 🤔

Reply:
At least the one before on this package it would seem

Reply:
I probably signed it off